src/Notifications/NotificationCenterController.cs

```diff
@@ -46,9 +56,45 @@ internal NotificationCenterController(DynamoView view, DynamoLogger dynLogger)
                 HorizontalOffset = notificationPopupHorizontalOffset,
                 VerticalOffset = notificationPopupVerticalOffset
             };
+
             notificationUIPopup.webView.EnsureCoreWebView2Async();
             notificationUIPopup.webView.CoreWebView2InitializationCompleted += WebView_CoreWebView2InitializationCompleted;
             logger = dynLogger;
+
+            RequestNotifications();
+        }
+
+        private void WebView_NavigationCompleted(object sender, Microsoft.Web.WebView2.Core.CoreWebView2NavigationCompletedEventArgs e)
+        {
+            AddNotifications(notificationsModel.Notifications);
```

Comment:
We make sure navigation is completed before calling the js function

Reply:
Is this because otherwise the js function may not exist?

Reply:
Yes, the navigation completed subscription makes sure the page content is loaded.